osu.Game.Rulesets.Catch/UI/CatchPlayfieldAdjustmentContainer.cs

```diff
@@ -10,15 +10,21 @@ namespace osu.Game.Rulesets.Catch.UI
 {
     public class CatchPlayfieldAdjustmentContainer : PlayfieldAdjustmentContainer
     {
+        private const float playfield_size_adjust = 0.8f;
```

Comment:
If we're now sharing this constant with `OsuPlayfieldAdjustmentContainer`, maybe it should be somewhere common?

Reply:
Yeah, not sure about this just yet. I think keeping it local is fine for now since it's only two rulesets which use this.